aggregator_core/src/datastore/tests.rs

```diff
@@ -1690,7 +1690,7 @@ async fn roundtrip_aggregation_job(ephemeral_datastore: EphemeralDatastore) {
         random(),
         dummy::AggregationParam(23),
         batch_id,
-        Interval::new(OLDEST_ALLOWED_REPORT_TIMESTAMP, Duration::from_seconds(1)).unwrap(),
+        Interval::new(OLDEST_ALLOWED_REPORT_TIMESTAMP, *task.time_precision()).unwrap(),
```

Comment:
not sure if it's cleaner to use `TIME_PRECISION` or the actual task value here. But it's a test...

Reply:
Better to use `task.time_precision()` I think.